homeassistant/components/flux/strings.json

```diff
@@ -0,0 +1,37 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Flux will update the temperature and brightness of your lights based on the time of day. During the day they will be bright, and they will gradually dim and fade to a red/orange at night. It will only affect lights that are turned on and listed below.",
+        "data": {
+          "name": "A name for your flux configuration",
+          "lights": "Select the lights you want controlled by Flux"
+        }
+      }
+    },
+    "error": {
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    }
+  },
+  "options": {
+    "flow_title": "Flux Options",
+    "step": {
+      "init": {
+        "description": "Flux will update the temperature and brightness of your lights based on the time of day. During the day they will be bright, and they will gradually dim and fade to a red/orange at night. It will only affect lights that are turned on and listed below.",
+        "data": {
+          "name": "A name for your flux configuration",
+          "lights": "Select the lights you want controlled by Flux",
+          "start_time": "Start time. If left empty this will be automatically calculated based on the location.",
```

Comment:
This, and several of the others, are too long and chatty for a label, please split between `data` and `data_description`. Also, if *automatically calculated based on the location* means it's calculated according to sunset and sunrise, please write that instead.